packages/expo-modules-core/CHANGELOG.md

```diff
@@ -12,6 +12,10 @@
 
 ### 💡 Others
 
+### ⚠️ Notices
+
+- Logger changes. ([#18251](https://github.com/expo/expo/pull/18251) by [@douglowder](https://github.com/douglowder))
```

Comment:
Making it available for use in Objective-C is a new feature, not a notice. Notices are intended to be used for deprecations. Please read [Updating Changelogs](https://github.com/expo/expo/blob/main/guides/contributing/Updating%20Changelogs.md).

Reply:
Actually I intended to remove all the objc annotations... I'll fix that